src/core/AutoRest.Core/Properties/Resources.resx

```diff
@@ -325,7 +325,7 @@ Modelers:
     <value>Parameters "subscriptionId" and "api-version" are not allowed in the operations section, define these in the global parameters section instead</value>
   </data>
   <data name="ServiceDefinitionParametersMissingMessage" xml:space="preserve">
-    <value>Parameter "{0}" is referenced but not defined in the global parameters section of Service Definition</value>
+    <value>Parameters "subscriptionId" and "api-version" must be defined in the parameters section in the swagger for the Azure Management Plane service.</value>
```

Comment:
minor: I think we could also say for "Azure Resource Management service", not sure what's best, so feel free to pick